stack-graphs/Cargo.toml

```diff
@@ -24,6 +24,7 @@ controlled-option = "0.4"
 either = "1.6"
 fxhash = "0.2"
 libc = "0.2"
+lsp-positions = { version="0.1", path="../lsp-positions" }
```

Comment:
What is the benefit of specifying the version of a crate inside the same repo explicitly? It cannot be anything but the version that is specified there anyway?

Reply:
The version information is used for the package snapshots that are uploaded to crates.io.  That way consumers who depend on `stack-graphs` will make sure to download a compatible version of `lsp-positions`, too.

Reply:
That is good to know!